tests/redirects/test_redirects.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+import pytest
+
+from utils.urls import assert_valid_url
+from .map_301 import URLS as REDIRECT_URLS
+from .map_301 import GITHUB_IO_URLS
+from .map_301 import MOZILLADEMOS_URLS
+
+# while these test methods are similar, they're each testing a
+# subset of redirects, and it was easier to work with them separately.
+@pytest.mark.headless
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('url', REDIRECT_URLS)
+def test_redirects(url):
+    url['base_url'] = "https://developer.mozilla.org"
```

Comment:
This should match [bedrock](https://github.com/mozilla/bedrock/blob/master/tests/redirects/test_redirects.py#L18-L19), using the ``base_url`` py.test fixture:

```
def test_redirects(url, base_url):
    url['base_url'] = base_url
```

You can then test it against staging:

```
py.test tests/redirects -vv --base-url https://developer.allizom.org --pdb
```
or the local Docker:

```
py.test tests/redirects -vv --base-url http://localhost:8000 --pdb
```

The ``--pdb`` drops you into the [Python debugger](https://docs.python.org/2/library/pdb.html), where you can print out the expected and actual values ("q" and Enter to quit).  If we were on py.test 3.x, the assets may be more useful (see PR #4210), but w/ py.test 2.x, you have to do your own test debugging.